src/Embed.php

```diff
@@ -10,23 +10,27 @@ class Embed
     private $url;
     private $key;
 
-    public $border = true;
-    public $title = false;
+    public $border;
+    public $title;
     public $theme;
 
-    public $width = '100%';
-    public $height = '800';
+    public $width;
+    public $height;
 
     /**
      * Default constructor
      *
      * @param $url string base url for the Metabase installation
      * @param $key int secret Metabase key
      */
-    public function __construct($url, $key)
+    public function __construct($url, $key, $border = true, $title = false, $width = '100%', $height = '800')
```

Comment:
Consider ordering the parameters so the most often used ones are first - perhaps: title, width, height, border